proxy/envoy/testdata/ingress-route-2.yaml.golden

```diff
@@ -0,0 +1,13 @@
+destination: foo.default.svc.cluster.local
+match:
+  http_headers:
```

Comment:
I think this PR has already had 50+ reviews with most of them being super pointless discussions, while PRs with bugs (or those that don't pass tests) have been happily force merged. So let's shelve these cosmetic nits and try to get the feature working first.